src/Avalonia.Diagnostics/Diagnostics/ViewModels/ClrPropertyViewModel.cs

```diff
@@ -9,6 +9,12 @@ internal class ClrPropertyViewModel : PropertyViewModel
         private Type _assignedType;
         private object? _value;
         private readonly Type _propertyType;
+        private enum PinnedStatus
```

Comment:
I think this enum can go away. Still, you shouldn't have two of the exact same enum defined.